feat: add pscredential and securestring (#283)
vmware/powershell-module-for-vmware-cloud-foundation: docs/documentation/functions/authentication/Connect-CloudBuilder.md

### Example 1

```powershell
Connect-CloudBuilder -fqdn sfo-cb01.sfo.rainpole.io -username admin -password VMw@re1!
```

> This example shows how to connect to the specified VMware Cloud Builder instance.

Reviewer: ```suggestion
This example shows how to connect to the specified VMware Cloud Builder instance using a clear-text username and password.

```

### Example 2

```powershell
$secureString = Read-Host -AsSecureString 'Password'
Connect-CloudBuilder -fqdn sfo-cb01.sfo.rainpole.io -username admin -password $secureString
```

> This example shows how to connect to the specified VMware Cloud Builder instance.

Reviewer: ```suggestion
This example shows how to connect to the specified VMware Cloud Builder instance by providing the password as a `SecureString`.

```

### Example 3

```powershell
Connect-CloudBuilder -fqdn sfo-cb01.sfo.rainpole.io -username admin
```

> This example shows how to connect to the specified VMware Cloud Builder instance.
> The operator will be prompted for a password.

Reviewer: ```suggestion
This example shows how to connect to the specified VMware Cloud Builder instance.

The user will be prompted for a password.

```

### Example 4

```powershell

> $credential = Get-credential

Reviewer: ```suggestion
$credential = Get-Credential

```

Connect-CloudBuilder -fqdn sfo-cb01.sfo.rainpole.io -credential $credential
```

> This example shows how to connect to the specified VMware Cloud Builder instance.

Reviewer: ```suggestion
This example shows how to connect to the specified VMware Cloud Builder instance using a PSCredential.

```

### Example 5

```powershell
Connect-CloudBuilder -fqdn sfo-cb01.sfo.rainpole.io
```

> This example shows how to connect to the VMware Cloud Builder instance.
> The operator will be prompted for a username and password.

Reviewer: ```suggestion
This example shows how to connect to the VMware Cloud Builder instance.

The user will be prompted for a username and password.

```

### -password

> The password to authenticate to the VMware Cloud Builder instance. 
> This parameter takes either a string or a SecureString variable.
> If not specified, a SecureString variable will be prompted for.

Reviewer: ```suggestion
The password to authenticate to the VMware Cloud Builder instance. 

This parameter takes either a string or a `SecureString` value.
If not specified, the user will be prompted for the `SecureString` value.

```

Accept wildcard characters: False
```

### -credential

> Specifies a user account to authenticate to the SDDC Manager instance.

Reviewer: ```suggestion
Specifies to authenticate to the SDDC Manager instance using a PSCredential.

```
